README.md

```diff
@@ -1 +1,54 @@
-# GatorGrade
+# GatorGrade: A Python Tool to Implement GatorGrader
+
+GatorGrade is a Python tool that executes GatorGrader, an automatic grading tool that can be used to check assignments through user-created checks. GatorGrade is the newer Python-based version of GatorGradle which can be found on GitHub at this link: https://github.com/GatorEducator/gatorgradle/blob/master/README.md. GatorGrade takes in a gatorgrader.yml file and tests each check for correctness against the code that it is connected to.
```

Comment:
This line is really long and could likely cause failures in the linter. Can you please break it down.